Evaluation/Input/Content/Section2.2_Data.md

```
@@ -0,0 +1,98 @@
+| CYP2D6 K<sub>m</sub> -> dxt   | µmol/L   |   4.65     | [Brown 2007](#main-references)| Michaelis Menten constant|
+| CYP2D6 K<sub>cat</sub> -> dxt (PM) | 1/min |  0       | [Brown 2007](#main-references)              | Catalytic rate constant                        |
+| CYP3A4 K<sub>m</sub>          | µmol/L   |   176.80   | [Lutz 2012](#main-references)              | Michaelis Menten constant                       |
+  
+### In vitro / physico-chemical Data Dextrorphan <a id="invitro-and-physico-chemical-data-dextrorphan"></a>
+
+A literature search was performed to collect available information on physicochemical properties of Dextrorphan. The obtained information from literature is summarized in the table below and is used for model building. 
+
+| **Parameter**| **Unit** | **Value** | **Source** | **Description**|
+| :-----------------------------| -------- | ---------- | ------------------------------------------ | ----------------------------------------------- |
+| MW                            | g/mol    |   257.37   | [HMDB-a](#main-references),[Wishart 2018](#main-references)           | Molecular weight                                |
+| logP|          |  1.38    | [Spaggiari 2014](#main-references)         | Partition coefficient between octanol and water|
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)